<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs</class>
 <widget class="QDialog" name="Prefs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxColoredSliders">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>401</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Sliders and Knobs color vary with value</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnChanNumb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>AXE-FX MIDI channel</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxDisplayBlownUp">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>471</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>When param value changes display Blown-up window</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxAutoConnect">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>MIDI auto-connect (AXE-FX, BCR2000, BCF2000, ...)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spnNumberInterval">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbxMIDIdataInterval">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>451</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>min MIDI data interval for CCs received from controllers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>X 0.05 ms  = </string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>471</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="ledInterval">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>70</width>
      <height>29</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <zorder>frame</zorder>
  <zorder>buttonBox</zorder>
  <zorder>cbxColoredSliders</zorder>
  <zorder>spnChanNumb</zorder>
  <zorder>label</zorder>
  <zorder>cbxDisplayBlownUp</zorder>
  <zorder>cbxAutoConnect</zorder>
  <zorder>spnNumberInterval</zorder>
  <zorder>cbxMIDIdataInterval</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Prefs</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Prefs</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
